spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -26,15 +26,15 @@
   # TODO Consider using fake_* spec helpers for examples below
 
   it "can validate none attestation" do
-    original_challenge = seeds[:security_key][:credential_creation_options][:challenge]
-    response = seeds[:security_key][:authenticator_attestation_response]
+    original_challenge = fake_challenge
+    encoded_challenge = WebAuthn::Utils.ua_encode(original_challenge)
 
     response = WebAuthn::AuthenticatorAttestationResponse.new(
-      attestation_object: response[:attestation_object],
-      client_data_json: response[:client_data_json]
+      attestation_object: encoded_fake_attestation_object,
+      client_data_json: encoded_fake_client_data_json(challenge: original_challenge)
     )
 
-    expect(response.valid?(original_challenge, "http://localhost:3000")).to eq(true)
+    expect(response.valid?(encoded_challenge, "http://localhost")).to eq(true)
     expect(response.credential.id.length).to be >= 16
   end
```

Comment:
Sounds like this example ended-up being almost exactly the same as `"is valid if everything's in place"`... What about deleting this one?

Reply:
Yeap, sounds reasonable 👍